client/src/layout/menu.js

```diff
@@ -192,6 +191,11 @@ export function fetchMenu(options = {}) {
                 title: _l("Interactive Tours"),
                 url: "tours",
             },
+            {
+                title: _l("Galaxy Version: " + Galaxy.config.version_major),
+                url: options.release_url + Galaxy.config.version_major + "_announce.html",
```

Comment:
```suggestion
                url: options.release_url + Galaxy.config.version_major + "_announce_user.html",
```

maybe should be the user-facing release notes, more interesting for users.

Reply:
Good call, thanks!